frontend/src/features/mapSideBar/tabs/ltsa/LtsaOwnershipInformationTitleOwnerForm.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { Formik } from 'formik';
+import { LtsaOrders } from 'interfaces/ltsaModels';
+import { noop } from 'lodash';
+import { render, RenderOptions } from 'utils/test-utils';
+
+import LtsaOwnershipInformationTitleOwnerForm, {
+  ILtsaOwnershipInformationTitleOwnerFormProps,
+} from './LtsaOwnershipInformationTitleOwnerForm';
+import { mockLtsaResponse } from './LtsaTabView.test';
+
+describe('LtsaOwnershipInformationTitleOwnerForm component', () => {
+  const setup = (
+    renderOptions: RenderOptions &
+      ILtsaOwnershipInformationTitleOwnerFormProps & { ltsaData?: LtsaOrders } = {},
+  ) => {
+    // render component under test
+    const component = render(
+      <Formik onSubmit={noop} initialValues={renderOptions.ltsaData ?? {}}>
+        <LtsaOwnershipInformationTitleOwnerForm
+          nameSpace={renderOptions.nameSpace ?? 'titleOrders.0.orderedProduct.fieldedData'}
+        />
+      </Formik>,
+      {
+        ...renderOptions,
+      },
+    );
+
+    return {
+      component,
+    };
+  };
+  it('renders ownership information title owner form', () => {
```

Comment:
I think this is fine for now, in the future it is nice to have at least one test to sanity check some actual data on the form. Ie, if you pass in an owner, does the owner name display correctly, that sort of thing. The problem with snapshots is that it is easy to ignore failures since they are so verbose.

Reply:
Sure, I will take care of this in future stories when I work